skrifa/src/meta/metrics.rs

```diff
@@ -0,0 +1,434 @@
+//! Global font and glyph specific metrics.
+
+use read_fonts::{
+    tables::{glyf::Glyf, hmtx::LongMetric, hvar::Hvar, loca::Loca},
+    types::{BigEndian, GlyphId},
+    TableProvider,
+};
+
+use crate::NormalizedCoord;
+
+/// Metrics for a text decoration.
+#[derive(Copy, Clone, PartialEq, Default, Debug)]
+pub struct Decoration {
+    /// Offset to the top of the decoration from the baseline.
+    pub offset: f32,
+    /// Thickness of the decoration.
+    pub thickness: f32,
+}
+
+/// Extents of a rectangular region.
+#[derive(Copy, Clone, PartialEq, Default, Debug)]
+pub struct BoundingBox {
+    /// Minimum x-coordinate.
+    pub x_min: f32,
+    /// Minimum y-coordinate.
+    pub y_min: f32,
+    /// Maximum x-coordinate.
+    pub x_max: f32,
+    /// Maximum y-coordinate.
```

Comment:
The comments here approach being noise.

Reply:
Moved this to `font_types::BoundingBox<T>` and made the comments less noisy. Hopefully this will be generally useful.